code/__HELPERS/roundend.dm

```diff
@@ -269,6 +269,20 @@
 
 	CHECK_TICK
 
+	//Process veteran achievements
+	for(var/client/C in GLOB.clients)
```

Comment:
As()

Reply:
Always be prepared for the client to be impossibly null.